### PR TITLE
fix: recreate kubectl pod

### PR DESCRIPTION
**Pull request by wansir**
Signed-off-by: hongming <talonwan@yunify.com>

### CODE REVIEW

File: pkg/models/iam/am.go [hunk 1]
```diff
@@ -636,6 +636,16 @@ func CreateClusterRoleBinding(username string, clusterRoleName string) error {
 	clusterRoleBindingLister := informers.SharedInformerFactory().Rbac().V1().ClusterRoleBindings().Lister()
 	found, err := clusterRoleBindingLister.Get(username)
```
Comment (runzexia) [inline pkg/models/iam/am.go]: handle error

File: pkg/models/iam/im.go [hunk 2]
```diff
@@ -797,10 +802,6 @@ func CreateUser(user *models.User) (*models.User, error) {
 		setAvatar(user.Username, user.AvatarUrl)
 	}
 
-	if err := kubeconfig.CreateKubeConfig(user.Username); err != nil {
```
Comment (runzexia) [inline pkg/models/iam/im.go]: if found.RoleRef.Name != constants.ClusterAdmin  => delete kubectl deploy?

File: pkg/models/iam/am.go [hunk 3]
```diff
@@ -657,11 +669,6 @@ func CreateClusterRoleBinding(username string, clusterRoleName string) error {
 			glog.Errorln("delete cluster role binding", err)
 			return err
 		}
-		if found.RoleRef.Name == constants.ClusterAdmin {
```
Comment (runzexia) [inline pkg/models/iam/am.go]: if found.RoleRef.Name != constants.ClusterAdmin => delete kubectl deploy?

Comment (wansir) [reply pkg/models/iam/am.go]: delete kubectl pod if cluster role is not cluster-admin, whether it exists or not

**Pull request merged**